hieroweb/web/src/hiero/web/InitialObject.java

```diff
@@ -2,34 +2,36 @@
 
 import org.checkerframework.checker.nullness.qual.NonNull;
 import org.hiero.sketch.dataset.LocalDataSet;
-import org.hiero.sketch.dataset.api.IJson;
+import org.hiero.sketch.dataset.ParallelDataSet;
+import org.hiero.sketch.dataset.api.IDataSet;
 import org.hiero.sketch.table.Table;
 import org.hiero.sketch.table.api.ITable;
 
 import javax.websocket.Session;
+import java.util.ArrayList;
 
 public class InitialObject extends RpcTarget {
-    // Used as a reply for loadTable.
-    static class RemoteTableId implements IJson {
-        public final String id;
-        RemoteTableId(String id) { this.id = id; }
-    }
-
-    public InitialObject(@NonNull String objectId) {
-        super(objectId);
-    }
-
     @HieroRpc
     void loadTable(@NonNull RpcRequest request, @NonNull Session session) {
         // TODO: look at request.  Now we just supply always the same table
-        String id = this.server.freshId();
         Table t = Table.testTable();
-        LocalDataSet<ITable> data = new LocalDataSet<ITable>(t);
-        TableTarget table = new TableTarget(id, data);
-        this.server.addObject(table);
+        final int parts = 5;
```

Comment:
This still looks like test/demo code, but we don't have the plumbing to do the end-to-end test yet. I'd just mark this method with a TODO.